lib/commands/description.js

```diff
@@ -17,6 +20,8 @@ Program
     .option('-u, --username <username>', 'The shopware username.')
     .option('-p, --plugin <plugin>', 'The name of the plugin, for which the description shall be updated.')
     .option('-l, --locale <locale>', 'The locale, for which the description shall be udpated, e.g. de_DE, en_GB etc.')
+    .option('--no-backup', 'Prevent creating a backup file for the old description text')
+    .option('--patch', 'Show diff and interactively ask before updating')
```

Comment:
I think this option should be named `interactive` since the changelog is always _patched_ if it's not empty.

Reply:
This is not `git` though, but a tool for automating processes that are tedious to be done by hand. Using `interactive` would then explain that it's not fully automatic but contains interactive steps.

Reply:
Now supports both :)